package/CHANGELOG

```diff
@@ -174,6 +174,8 @@ Enhancements
     checking if it can be used in parallel analysis. (Issue #2996, PR #2950)
 
 Changes
+  * `contacts.Contacts` now stores data using the `Contacts.results.timeseries`
```

Comment:
Please add entry to deprecations

Reply:
Sorry, should have done this when merging

Reply:
I was actually just filling in all these bits as you wrote the review 🤣

Reply:
oops, sorry, just committed the change